hack/kind-cluster-build.sh

```diff
@@ -14,10 +14,10 @@ Options:
        -h,--help               prints the usage message
        -n,--name               name of the Kubernetes cluster,default value: kind
        -c,--nodeNum            the count of the cluster nodes,default value: 3
-       -k,--k8sVersion         version of the Kubernetes cluster,default value: v1.17.2
+       -k,--k8sVersion         version of the Kubernetes cluster,default value: v1.15.6
```

Comment:
why do we change to v1.15.6?

Reply:
According to https://kubernetes.io/blog/2019/07/18/api-deprecations-in-1-16/, there would exist incompatible things between helm >= 2.14 and kubernetes >= 1.16.0

In order to avoid this problem, I suggest changing the default version for kubernetes as  1.15 

FYI: https://github.com/helm/helm/issues/6374